composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/console": "~2.8|~3.0|~4.0",
         "symfony/config": "~2.8|~3.0|~4.0",
         "symfony/yaml": "~2.8|~3.0|~4.0",
-        "vlucas/phpdotenv": "^2.4"
+        "josegonzalez/dotenv": "2.*"
```

Comment:
Why not semver?

Reply:
I picked this version to be consistent with the version used in cakephp/app.